### PR TITLE
refactor: Make login headline one line and add auth method section

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Screen Shot 2022-05-31 at 12 58 03

### CODE REVIEW

**Review by greyscaled — APPROVED** [review]

File: site/src/components/SignInForm/SignInForm.tsx [hunk 1]
```diff
@@ -119,13 +143,30 @@ export const SignInForm: FC<SignInFormProps> = ({
...
-          <Link href={`/api/v2/users/oauth2/github/callback?redirect=${encodeURIComponent(redirectTo)}`}>
-            <Button disabled={isLoading} fullWidth type="submit" variant="contained">
-              {Language.githubSignIn}
-            </Button>
-          </Link>
-        </div>
+        <>
+          <div className={styles.divider}>
+            <div className={styles.dividerLine} />
+            <div className={styles.dividerLabel}>Or sign in with</div>
```
Comment (kylecarbs) [inline site/src/components/SignInForm/SignInForm.tsx:149]: What does this look like with `or`? `sign in` is a bit duplicative, but it's not bad.

**Review by kylecarbs — APPROVED** [review]

File: site/src/components/SignInForm/SignInForm.tsx (hunk 1 above)
Comment (BrunoQuaresma) [reply site/src/components/SignInForm/SignInForm.tsx:149]: Screen Shot 2022-05-31 at 13 04 05
Screen Shot 2022-05-31 at 13 04 23

Comment (BrunoQuaresma) [reply site/src/components/SignInForm/SignInForm.tsx:149]: Feel free to suggest a diff copy for this directly on GitHub so I can just merge it in one click.

Comment (kylecarbs) [reply site/src/components/SignInForm/SignInForm.tsx:149]: Good idea!

Comment (kylecarbs) [inline site/src/components/SignInForm/SignInForm.tsx:149]:
```suggestion
            <div className={styles.dividerLabel}>Or</div>
```

**Pull request merged**